### ISSUE TITLE
Add monochrome icon

### ISSUE THREAD

**Issue by xmha97** [rkkr/simple-keyboard#426]
Please support the Android 13 Themed App Icons feature.

![Screenshot_2023-11-03-18-58-58-321_com miui home](https://github.com/rkkr/simple-keyboard/assets/23485114/a37b0804-2008-47fe-b886-d366170503b3)

**Comment by geesquer4u** [comment]
Yes. Please add that. Its a nice extra touch to this nice keyboard

**Comment by SpiritOTHawk** [comment]
Yes, needed

**Comment by SpiritOTHawk** [comment]
<img width="1080" height="2400" alt="Image" src="https://github.com/user-attachments/assets/bfff0bba-c8f6-4e04-a20f-35e5939c0837" />
